### PR TITLE
LF Tag for Sage Journal and Taylor

### PR DESCRIPTION
**Pull request by socheres**
https://github.com/ubtue/DatenProbleme/issues/1397

### CODE REVIEW

File: SAGE Journals.js [hunk 1]
```diff
@@ -198,6 +198,10 @@ function scrape(doc, url) {
 					if (tags) item.tags = tags.map(n => n.textContent);
 				}
 			}
+			// mark articles as "LF" (MARC=856 |z|kostenfrei), that are published as open access
+			let openAccessTag = doc.querySelector('.accessIcon');Z.debug(openAccessTag)
+			let altLabel = openAccessTag.alt;Z.debug(altLabel)
+			if (openAccessTag && altLabel.match(/open\saccess/gi)) item.notes.push({note: 'LF:'});
```
Comment (jriedl) [inline SAGE Journals.js:204]: Please remove the debug statements. 
Access to attributes should always make make sure that the selected element exists beforehand, so
```javascript
let openAccessTag = doc.querySelector('.accessIcon');
let altLabel = openAccessTag && openAccessTag.hasAttribute("alt") ? openAccessTag.alt : "";
if (altLabel.match(/open\s+access/gi) ...
```
Alternatively, and maybe more concise would be (_beware_: naming applies to T&F) 
```javascript
let AccessIconLocation = doc.querySelector('.accessIconLocation[alt]');
if (AccessIconLocation && AccessIconLocation.alt.match(/open\s+access/gi))
     item.notes.push('LF:');
```

File: Taylor and Francis+NEJM.js [hunk 2]
```diff
@@ -182,6 +182,11 @@ function finalizeItem(item, doc, doi, baseUrl) {
 	//deduplicate
 	item.notes = Array.from(new Set(item.notes.map(JSON.stringify))).map(JSON.parse);
 	
+	// mark articles as "LF" (MARC=856 |z|kostenfrei), that are published as open access
+	let openAccessTag = doc.querySelector('.accessIconLocation');
+	let altLabel = openAccessTag.alt;
+	if (openAccessTag && altLabel.match(/open\saccess/gi)) item.notes.push('LF:');
+
```
Comment (jriedl) [inline Taylor and Francis+NEJM.js:189]: See above

File: SAGE Journals.js (hunk 1 above)
Comment (socheres) [reply SAGE Journals.js]: Thanks for the code review. I choose the alternative solution.

**Comment by socheres** [comment]
Should the tag "LF:" pushed to key-value pair like `note: LF:`?

**Comment by jriedl** [comment]
Sorry, I don't understand the question.

**Comment by socheres** [comment]
there are two ways, how i can push the note "LF" as key-value as nested object notes.note `item.notes.push({note: 'LF:'});` (SAGE) or simply 
`item.notes.push('LF:');`(T&F) in case that there are other note tags like orcid etc.

**Comment by jriedl** [comment]
PR already merged and installed on nu.
Approach should be analogous with the working example Brill translator, i.e. `if (openAccessTag) item.notes.push('LF:');`

**Pull request merged**